paper/tex/methods.tex

```diff
@@ -170,74 +171,97 @@ \subsubsection{GPz}
+Thus the \pz\ posterior $p(\mathbf{\hat{F}} \vert z, T_i) = \int p(\mathbf{\hat{F}} \vert \mathbf{F}) p(\mathbf{F} \vert z, z_j, \mathbf{\hat{F}}_j) d\mathbf{F}$ may be interpreted as the probability that the training and the target galaxies have the same SED at different redshifts.
+The flux prediction $p(\mathbf{F} \vert z, z_j, \mathbf{\hat{F}}_j)$ of the training galaxy at redshift $z$ is modeled via the Gaussian process described above; more detail is provided in \citet{Leistedt:17}.
+
+In this study, the default settings of \delight\ were used, with the exception that the PDF bins were set to be linearly-spaced rather than logarithmic.
+The Gaussian process was trained using the full DC1 training set.
+We used the full DC1 template set with a flat prior in magnitude and SED type.
+Photometric uncertainties from the inputs are propagated into the code, while non-detections for each band are set to the mean of the respective bands.
+
+\subsection{trainZ: a pathological \pz\ estimator}
+\label{sec:trainz}
+
+We also consider a pathological \pzpdf\ estimation method, dubbed \trainz, which assigns each test set galaxy a \pzpdf\ equal to the normalized redshift distribution $N(z)$ of the training set, according to
 \begin{equation}
-p(z) = \frac{1}{N_{ \mathrm train}}\sum_{\mathrm i=1}^{N_{\mathrm train}}z_{\mathrm train}
+	p(z \vert \{z_{j}\}) \equiv \frac{1}{N_{ \mathrm train}}\sum_{\mathrm i=1}^{N_{\mathrm train}} \begin{cases} 1 & \text{if\ } z_{k}\leq z_{i} < z_{k+1}\\ 0 & \text{otherwise} \end{cases}.
 \end{equation}
-That is, we simply set the redshift PDF of every galaxy equal to the normalized $N(z)$ of the training sample.  This estimator is essentially a k nearest-neighbour estimator with k equal to the number of galaxies in the training sample.  As the training sample is drawn from the same underlying distribution as the test sample, modulo small deviations due to sample size, the quantiles of the training and test distributions should be identical, modulo fluctuations due to finite sample size.  This is a wildly unrealistic estimator, as it assigns all galaxies, no matter their apparent magnitude, colour, or true redshift, the same redshift PDF, and is thus uninformative at the level of individual object redshifts, but is designed to perform very well for the ensemble of all objects.  If the training set was not representative, this estimator would produce biased results, and any attempts to break up the sample into tomographic bins will fail, as every galaxy has an identical $p(z)$.  We will discuss this method and cautions relative to metrics in Section~\ref{sec:caution}.
+Unlike the other methods, the \trainz\ estimator is \textit{independent of the photometric data}, effectively performing a KNN procedure with $k=N_{\rm train}$.
```

Comment:
I'd add back in "That is, every galaxy is assigned an identical p(z), equal to the N(z) of the training data", just to be more explicit as to what trainZ is doing.

Reply:
Is that not the same as the sentence above the equation?